apps/portals-base/src/components/TabbedSynapseObjects.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
```

Comment:
Formatting + import changes cause diff to treat this as a new file, but it's not.